packages/svelte/src/internal/client/dom/blocks/svelte-html.js

```diff
@@ -0,0 +1,62 @@
+import { render_effect, teardown } from '../../reactivity/effects.js';
+import { set_attribute } from '../elements/attributes.js';
+import { set_class } from '../elements/class.js';
+import { hydrating } from '../hydration.js';
+
+/**
+ * @param {() => Record<string, any>} get_attributes
+ * @returns {void}
+ */
+export function svelte_html(get_attributes) {
+	const node = document.documentElement;
+	const own = {};
+
+	/** @type {Record<string, Array<[any, any]>>} to check who set the last value of each attribute */
+	// @ts-expect-error
+	const current_setters = (node.__attributes_setters ??= {});
+
+	/** @type {Record<string, any>} */
+	let attributes;
+
+	render_effect(() => {
+		attributes = get_attributes();
+
+		for (const name in attributes) {
+			const current = (current_setters[name] ??= []);
+			const idx = current.findIndex(([owner]) => owner === own);
+			const old = idx === -1 ? null : current.splice(idx, 1)[0][1];
```

Comment:
never understood the point of this abbreviation, it takes two syllables and turns it into three 😆 

```suggestion
			const index = current.findIndex(([owner]) => owner === own);
			const old = index === -1 ? null : current.splice(index, 1)[0][1];
```